Shared/Flows/Onboarding/UserOnboardingViewController.swift

```diff
@@ -31,7 +31,8 @@ class UserOnboardingViewController: ViewController {
         self.view.addSubview(self.messageContent)
         self.messageContent.configureBackground(color: ThemeColor.D1.color,
                                                 textColor: ThemeColor.T3.color,
-                                                brightness: 1.0,
+                                                brightness: 1,
+                                                focusAmount: 1,
```

Comment:
Message content now has a focus amount parameter which indicates how in focus the whole stack is.